stripe_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func TestBearerAuth(t *testing.T) {
-	c := &stripe.BackendConfiguration{URL: stripe.APIURL}
+	c := stripe.GetBackend(stripe.APIBackend).(*stripe.BackendConfiguration)
```

Comment:
Dang it. I kind of punted on finding the right way to solve this for now.

It's an improvement that we now use `GetBackend` to get a default backend with good values set, but it's still bad that we then need to cast it back to `BackendConfiguration`. The problem is that each of these tests uses `BackendConfiguration.NewRequest`, and that is a function that's not part of the `Backend` interface.

We should either add it to the interface, or find a better way to write these tests, with a preference for the latter.